manifests/params.pp

```diff
@@ -123,28 +123,24 @@
     'Suse': {
       case $::operatingsystem {
         'OpenSuSE': {
-          if versioncmp( $::operatingsystemmajrelease, '12' ) >= 0 {
```

Comment:
12 is the smallest version we support (we support 12 and 15) so no need for the comparison.